packages/in_app_purchase/lib/src/billing_client_wrappers/purchase_wrapper.g.dart

```diff
@@ -18,6 +18,18 @@ PurchaseWrapper _$PurchaseWrapperFromJson(Map json) {
       originalJson: json['originalJson'] as String);
 }
 
+Map<String, dynamic> _$PurchaseWrapperToJson(PurchaseWrapper instance) =>
```

Comment:
nit: We're generating a bunch of `toJson` methods we don't need now. It's not a big deal but I wonder if there's a way to selectively enable this generation instead.

Reply:
Discussed offline and we will keep the methods there for now.